### PR TITLE
ACE_Main major cleanup

### PR DESCRIPTION
**Pull request by esteldunedain**
Cleanup everything not used on ACE_Main.

For the many macros removed, I'm pretty sure none of those removed are used, did quick searches + limited testing in game. Ideally all devs should take a couple of minutes to go over the diff and check they aren't using any of them.

### CODE REVIEW

**Comment by KoffeinFlummi** [comment]
:+1:

File: addons/main/CfgModuleCategories.hpp [hunk 1]
```diff
@@ -1,8 +0,0 @@
-class CfgFactionClasses {
-    class NO_CATEGORY;
-    class ACE: NO_CATEGORY {
```
Comment (Glowbal) [inline addons/main/CfgModuleCategories.hpp]: This is used by our modules

Comment (esteldunedain) [inline addons/main/CfgModuleCategories.hpp]: Ok, I'll revert this

**Comment by Glowbal** [comment]
:+1:

**Comment by jaynus** [comment]
Nothing stood out to me looks good

**Pull request merged**